scripts/RNA_analysis/analysis_report.py

```diff
@@ -665,6 +665,8 @@ def main(project_id,sample_names,single_end,config_file,Map_Stat,Read_Dist,FPKM,
     help = "Samplenames should be given as a coma delimited string. Default will be the samples given by the tophat_out - directories")
     parser.add_option( "-b", "--complexity", dest="complexity",action="store_true", default=False,
     help = "to include library complexity plot")
+    parser.add_option('-G', '--genome', action="store", dest="genome", default=None,
+    help="Reference genome name (eg. 'GRCh37') Default: fetch from status DB")
```

Comment:
Is there a list of supported species? Surely it should work with any reference genome?

But if wanted, you could use the dict I mentioned above here too: `species.keys()` in the help text.